app/main/views/audits.py

```diff
@@ -26,6 +40,20 @@ def list_audits():
         else:
             abort(400, "Invalid audit type")
 
+    acknowledged = request.args.get('acknowledged', None)
+    if acknowledged:
+        if is_valid_acknowledged_state(acknowledged):
+            if acknowledged == 'acknowledged':
```

Comment:
Is it worth using `dmutils.config.convert_to_boolean` here? `&acknowledged=acknowledged` doesn't look very nice